xmppserver/changelog.html

```diff
@@ -162,6 +162,13 @@ <h1>Openfire Changelog</h1>
     
     <div id="pageBody">
 
+<h2>(Next version) <span style="font-weight: normal;">(release date)</span></h2>
+
+<h2>Improvement</h2>
+<ul>
+    <li>[<a href="https://igniterealtime.atlassian.net/browse/OF-2360">OF-2360</a>] - Plugin version information accessible through cluster manager</li>
+</ul>
+
```

Comment:
Unlike with plugins, we typically do _not_ update the changelog for Openfire when changes are made. Instead, we'll generate the entire changelog in one go when a release is made.

Reply:
👍